index.html

```diff
@@ -306,60 +325,60 @@ <h1 class="intro">
           you see and have a project you need coded, don't hestiate to contact
```

Comment:
- [ ] And for the `about` section I think you missed aligning the about and languages beside each other so please Fix that so you can use flex or grid to that to display them as a row, not as a column to match the design so kindly fix it.
and please stick with the design as much as possible font, size style and background color💪🏻💪🏻💪🏻

- Your design 
![image](https://user-images.githubusercontent.com/58553711/177784352-f7244494-ed94-4c50-98eb-9672c6d1fa79.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177784459-d5ad7ab3-7015-4809-b5d7-4c13ad00724a.png)